### PR TITLE
Update website

### PR DESCRIPTION
**Pull request by styrix560**

### CODE REVIEW

**Comment by styrix560** [comment]
The images (and dates) for the timeline are still missing.

File: public/locales/de/translation.json [hunk 1]
```diff
@@ -158,23 +158,38 @@
       },
       "links": {
         "wsi": "Skill 08 bei WorldSkills International",
-        "ws2022": "Skill 08 bei den WorldSkills 2022 SE"
+        "wsg": "Skill 08 bei den WorldSkills 2024 Lyon"
```
Comment (JonasWanke) [inline public/locales/de/translation.json:161]:
```suggestion
        "wsg": "Skill 08 bei den WorldSkills Lyon 2024"
```

File: public/locales/de/translation.json [hunk 2]
```diff
@@ -190,13 +205,38 @@
...
+          "text": "Die Gewinner des Albert-Einstein-Cups in Neubrandenburg sind: Thomas Gassmann (Schweiz), Glenn Skrzypczak (Deutschland) and Bence Lakos (Ungarn). Justin Konratt gewinnt die zusätzliche Disziplin Speed-Programming."
+        },
+        "wsc2022": {
+          "date": "13&nbsp;&thinsp;&ndash;&thinsp;16&nbsp;Oktober 2022",
+          "title": "WorldSkills Competition 2022 Special Edition",
+          "text": "Glenn Skrzypczak wird Vize-Weltmeister im Speed-Programming."
+        },
+        "euroskills2023": {
+          "date": "5&nbsp;September&thinsp;&ndash;&thinsp;9&nbsp;September 2023",
+          "title": "EuroSkills Competition 2023 Gdansk",
```
Comment (JonasWanke) [inline public/locales/de/translation.json:223]:
```suggestion
          "title": "EuroSkills Gdańsk 2023",
```

File: public/locales/de/translation.json [hunk 3]
```diff
@@ -190,13 +205,38 @@
...
+          "text": "Beide Skills der Deutschen Meisterschaft wurden von Daniel Kalisch, Simon Brebeck and Nils Jakubzick gewonnen. Florian Darsow ist bester im Speed-Programming."
+        },
+        "einsteinCup2024": {
+          "date": "27&nbsp;Mai&thinsp;&ndash;&thinsp;30&nbsp;Mai 2024",
+          "title": "Albert-Einstein-Cup",
+          "text": "Jongwon Kim (Südkorea), Daniel Kalisch (Deutschland) und [Schweiz] sind die Gewinner des Albert-Einstein-Cups. Stefan Kornel (Ungarn) gewinnt die zusätzliche Disziplin Speed-Programming. Der &ldquo;Hungarian Speed Test&ldquo; wurde das erste mal durchgeführt und von Nils Jakubzick (Deutschland) gewonnen."
+        },
+        "wsc2024": {
+          "date": "10&thinsp;&ndash;&thinsp;15&nbsp;September 2024",
+          "title": "WorldSkills Competition 2024 Lyon",
```
Comment (JonasWanke) [inline public/locales/de/translation.json:238]:
```suggestion
          "title": "WorldSkills Lyon 2024",
```

File: public/locales/de/translation.json [hunk 4]
```diff
@@ -158,23 +158,38 @@
...
+          "text": "Beide Skills der Deutschen Meisterschaft wurden von Daniel Kalisch, Simon Brebeck and Nils Jakubzick gewonnen. Florian Darsow ist bester im Speed-Programming."
+        },
+        "einsteinCup2024": {
+          "date": "27&nbsp;Mai&thinsp;&ndash;&thinsp;30&nbsp;Mai 2024",
+          "title": "Albert-Einstein-Cup",
+          "text": "Jongwon Kim (South Korea), Daniel Kalisch (Germany) und Wei-Hung Lai (Chinese Taipei) sind die Gewinner des Albert-Einstein-Cups. Stefan Kornel (Ungarn) gewinnt die zusätzliche Disziplin Speed-Programming. Der &ldquo;Hungarian Speed Test&ldquo; machte seinen ersten Auftritt und wurde von Nils Jakubzick (Deutschland) gewonnen."
+        },
+        "wsc2024": {
+          "date": "10&thinsp;&ndash;&thinsp;15&nbsp;September 2024",
+          "title": "WorldSkills Competition 2024 Lyon",
```
Comment (JonasWanke) [inline public/locales/de/translation.json:191]:
```suggestion
          "title": "WorldSkills Lyon 2024",
```

File: public/locales/en/translation.json [hunk 5]
```diff
@@ -157,23 +157,38 @@
...
+          "text": "Both skills of the German National Championship are won by Daniel Kalisch, Simon Brebeck and Nils Jakubzick. Speed-Programming is won by Florian Darsow."
+        },
+        "einsteinCup2024": {
+          "date": "27&nbsp;May&thinsp;&ndash;&thinsp;30&nbsp;May 2024",
+          "title": "Albert-Einstein-Cup",
+          "text": "Jongwon Kim (South Korea), Daniel Kalisch (Germany) and Wei-Hung Lai (Chinese Taipei) are the winners of the Albert-Einstein-Cup. Stefan Kornel (Hungary) wins the additional Speed-Programming discipline. The &ldquo;Hungarian Speed Test&ldquo; made its first appearance and was won by Nils Jakubzick (Germany)."
+        },
+        "wsc2024": {
+          "date": "10&thinsp;&ndash;&thinsp;15&nbsp;September 2024",
+          "title": "WorldSkills Competition 2024 Lyon",
```
Comment (JonasWanke) [inline public/locales/en/translation.json:190]:
```suggestion
          "title": "WorldSkills Lyon 2024",
```

File: public/locales/en/translation.json [hunk 6]
```diff
@@ -189,13 +204,38 @@
       },
       "links": {
         "wsi": "Skill 09 at WorldSkills International",
-        "ws2022": "Skill 09 at the WorldSkills 2022 SE"
+        "wsg": "Skill 09 at the WorldSkills 2024 Lyon"
```
Comment (JonasWanke) [inline public/locales/en/translation.json:207]:
```suggestion
        "wsg": "Skill 09 at the WorldSkills Lyon 2024"
```

File: public/locales/en/translation.json [hunk 7]
```diff
@@ -189,13 +204,38 @@
...
+          "text": "Glenn Skrzypczak becomes the vice world-champion in the additional Speed-Programming discipline."
+        },
+        "nationalChampionship2023": {
+          "date": "25&nbsp;May&thinsp;&ndash;&thinsp;25&nbsp;May 2023",
+          "title": "German National Championship",
+          "text": "Both skills of the German National Championship are won by Daniel Kalisch, Simon Brebeck and Nils Jakubzick. Speed-Programming is won by Florian Darsow."
+        },
+        "euroskills2023": {
+          "date": "5&nbsp;September&thinsp;&ndash;&thinsp;9&nbsp;September 2023",
+          "title": "EuroSkills Competition 2023 Gdansk",
```
Comment (JonasWanke) [inline public/locales/en/translation.json:227]:
```suggestion
          "title": "EuroSkills Gdańsk 2023",
```

File: public/locales/en/translation.json [hunk 8]
```diff
@@ -189,13 +204,38 @@
...
+          "text": "Glenn Skrzypczak achieves a gold medal at the EuroSkills and additionally wins the Speed-Programming."
+        },
+        "einsteinCup2024": {
+          "date": "27&nbsp;May&thinsp;&ndash;&thinsp;30&nbsp;May 2024",
+          "title": "Albert-Einstein-Cup",
+          "text": "Simon Brebeck (Germany), Lukas Hertkorn (Germany) and [Schweiz] are the winners of the Albert-Einstein-Cup. Stefan Kornel (Hungary) wins the additional Speed-Programming discipline. The &ldquo;Hungarian Speed Test&ldquo; made its first appearance and was won by Nils Jakubzick (Germany)."
+        },
+        "wsc2024": {
+          "date": "10&thinsp;&ndash;&thinsp;15&nbsp;September 2024",
+          "title": "WorldSkills Competition 2024 Lyon",
```
Comment (JonasWanke) [inline public/locales/en/translation.json:237]:
```suggestion
          "title": "WorldSkills Lyon 2024",
```

**Review by JonasWanke — CHANGES_REQUESTED** [review]

**Review by JonasWanke — APPROVED** [review]

**Pull request merged**